pkg/doc/did/doc.go

```diff
@@ -115,7 +115,7 @@ type PublicKey struct {
 
 	Value []byte
 
-	jsonWebKey *jose.JWK
+	JSONWebKey *jose.JWK `json:"-"`
```

Comment:
As discussed expose it through function keep the variable private, to maintain standards among all format

Reply:
I really don't understand it. Why should we hide it? What's the standard? It's defined in https://www.w3.org/TR/did-core/#public-keys, `publicKeyJwk` field. Why should we expose `Value` field and don't export `JSONWebKey` field? What's the rule here?

Reply:
+ @troyronda @fqutishat

Reply:
Some one who iterates through public key will be confused to use which one to use.

Reply:
I agree it's confusing to have both JSONWebKey and value/type exposed - these fields have similar purpose.

Reply:
Added `NewPublicKeyFromBytes()` and `NewPublicKeyFromJWK()` constructor functions. `jsonWebKey` field of `PublicKey` is hidden, added `JSONWebKey()` getter.